numpy/core/tests/test_datetime.py

```diff
@@ -26,6 +26,7 @@ class TestDateTime:
     def test_datetime_dtype_creation(self):
         for unit in ['Y', 'M', 'W', 'D',
                      'h', 'm', 's', 'ms', 'us',
+                     'μs',  # alias for us
```

Comment:
This assumes the file encoding is utf-8, which is the default. It might help some editors to explicitly add a 
`# -*- coding: utf-8 -*-` directive to the first line of the file.

Reply:
Well, all non-dead versions of python assume source files are utf-8 too, so arguably an up to date editor should be able to guess the correct encoding.

If you're worried, I can also use `\N{Greek small letter mu}` instead.

Reply:
no, it was just a nit anyway.